test/benchmark-crud/main.cpp

```diff
@@ -71,8 +71,8 @@ inline void erase(TableRef table, const std::vector<size_t> order)
 
 int main()
 {
-    const size_t target_size = 1100 * 1000L;
-    const int num_tables = 50;
+    const size_t target_size = 1100 * 100L;
```

Comment:
This decreases the height of the B+-tree from 3 to 2 levels. I think the 1100K entries was chosen specifically to get 3 levels, matching tables of sizes 1M to 1G.